packages/code-studio/src/styleguide/Alerts.tsx

```diff
@@ -16,7 +18,7 @@ class Alerts extends Component {
       Alerts.renderAlert(brand)
     );
     return (
-      <div>
+      <div {...sampleSectionIdAndClasses('alerts')}>
```

Comment:
I probably would've thought to put this in a component instead that just takes props `name` and `className` - think it would be clearer. `Section`, `SectionView`, `SectionFlex` would cover `div` vs. `View` vs `Flex` cases. Not really worth refactoring though.

Reply:
Do something like:
`await page.locator('selector').evaluate(element => element.style.opacity= '0');`